libkineto/src/Config.cpp

```diff
@@ -606,8 +606,7 @@ void Config::printActivityProfilerConfig(std::ostream& s) const {
   for (const auto& activity : selectedActivityTypes_) {
     activities.push_back(toString(activity));
   }
-  s << "  Enabled activities: " << fmt::format("{}", fmt::join(activities, ","))
-    << std::endl;
```

Comment:
FYI, clang-tidy perf checks tells you to avoid std::endl anyway because it unnecessarily flushes the ostream. We've already banned them upstream in PyTorch.